auto_withdraw_delegate.service

```diff
@@ -3,7 +3,8 @@ Description=Cosmos Systemd Auto Withdraw and Delegate
 After=network-online.target
 [Service]
 User=gaiad
-ExecStart=/usr/local/bin/auto-delegate.sh
+ExecStart=/bin/sh -c "KEY=$(systemd-ask-password "Enter your key password: ")"
```

Comment:
There is an error while running the code. 
<img width="852" alt="screen shot 2019-01-09 at 18 02 23" src="https://user-images.githubusercontent.com/3680497/50916318-1afe2500-143b-11e9-8a45-c90b3224674b.png">